resources/compass/values.yaml

```diff
@@ -15,8 +15,8 @@ global:
       dir: develop/
       version: "1ce3b6c3"
     healthchecker:
-      dir: develop/
-      version: "26601526"
+      dir: pr/
+      version: "PR-287"
```

Comment:
Please reset change for healthchecker image. Unfortunately we cannot have `PR-XXX` images here.